### PR TITLE
typo fix "onSucess"

### PR DESCRIPTION
**Pull request by lahosken**
createMember, loginMember had parameter typo "onSucess"

with this change, each method has two signatures: one with typo remaining, one with typo fixed

### CODE REVIEW

File: src/api/TokenIO.h [hunk 1]
```diff
@@ -67,6 +67,11 @@ globalRpcErrorCallback:(OnError)globalRpcErrorCallback;
  * @param alias member alias to use, must be unique
  */
 - (void)createMember:(Alias *)alias
+            onSuccess:(OnSuccessWithTKMember)onSuccess
+             onError:(OnError)onError;
+
+// invokes createMember:onSuccess:onError
+- (void)createMember:(Alias *)alias
             onSucess:(OnSuccessWithTKMember)onSuccess
              onError:(OnError)onError;
```
Comment (sibinlutoken) [inline src/api/TokenIO.h]: Why do we need to keep the typo apis?

Comment (lahosken) [reply src/api/TokenIO.h]: No reason, will remove.

**Comment by lahosken** [comment]
Since this change breaks an API, should I be bumping a version number? If the answer to that is "yes", my next question is: how do I do that? bin/publish, maybe?

**Comment by sibinlutoken** [comment]
the formal fix would be:
1. delete the old one
2. Add deprecation notice which will throw on old typo apis and point developer to new apis.

But for now, Im the only developer, so feel free just delete the typo one. Don't need to bump version for this small fix. usually version is bumped due to critical bug fix or new feature.

for bumping the version, bin/publish -p

**Pull request merged**